molecule/_shared/converge.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  become_user: root
+  tasks:
+    - name: "Include configure_nomad_server"
```

Comment:
Not sure why the reference to nomad is used here. Can you please clarify?

Reply:
A type, I have fixed